src/SOS/SOS.NETCore/CMakeLists.txt

```diff
@@ -22,5 +22,8 @@ if(NOT ${NUGET_PACKAGES} STREQUAL "")
         set(DIASYMREADER_ARCH amd64)
     endif()
 
-    install(FILES ${NUGET_PACKAGES}/microsoft.diasymreader.native/1.7.0/runtimes/win/native/Microsoft.DiaSymReader.Native.${DIASYMREADER_ARCH}.dll DESTINATION . )
+    # Until issue https://github.com/dotnet/diagnostics/issues/324 is done there is no arm64 version of the diasymreader in this package
+    if(NOT CLR_CMAKE_TARGET_ARCH_ARM64)
+        install(FILES ${NUGET_PACKAGES}/microsoft.diasymreader.native/1.7.0/runtimes/win/native/Microsoft.DiaSymReader.Native.${DIASYMREADER_ARCH}.dll DESTINATION . )
```

Comment:
Quick question: will the managed or native parts of SOS ever try to access this dll and fail to locate it in any context? Will we get a managed exception or an unhandled file not found for a native or full windows PDB?

Reply:
SOS should fail gracefully, but it will print out a error message that I might want to turn into a "ExtDbgOut" that is only displayed when debug out is turned on.